pennylane/tape/qscript.py

```diff
@@ -546,25 +551,51 @@ def trainable_params(self, param_indices):
 
         self._trainable_params = sorted(set(param_indices))
 
-    def get_operation(self, idx):
+    def get_operation(self, idx, return_op_index=False):
         """Returns the trainable operation, and the corresponding operation argument
         index, for a specified trainable parameter index.
 
         Args:
             idx (int): the trainable parameter index
-
+            return_op_index (bool): the trainable parameter index
```

Comment:
? `idx` is "the trainable parameter idx", so how is `return_op_index` also "the trianable parameter idx?

Reply:
The function also returns the operation index.